storage/ndb/rest-server/data-access-rondb/src/db-operations/pk/common.cpp

```diff
@@ -932,3 +937,47 @@ int GetByteArray(const NdbRecAttr *attr, const char **first_byte, Uint32 *bytes)
     return -1;
   }
 }
+
+bool CanRetryOperation(RS_Status status) {
+  bool retry = false;
+  if (status.http_code != SUCCESS) {
+    if (status.classification == NdbError::TemporaryError) {
```

Comment:
Have you ever actually come across TemporaryErrors? How can they happen?

Reply:
Overload on RonDB causes TimeOut Errors (common), node failure causes a number of different temporary errors (happens every now and then). Out of a specific resource error can cause temporary error (not very common).